src/cmd-generate-release-meta

```diff
@@ -217,6 +217,13 @@ def append_build(out, input_):
     if i is not None:
         arch_dict["media"]["metal"]["artifacts"].setdefault("pxe", {})["rootfs"] = artifact(i)
 
+    # Qemu s390x specific additions
+    i = input_.get("images", {}).get("secex-qemu", None)
+    if i is not None:
+        arch_dict["media"]["qemu"]["artifacts"]["secure-execution"] = {
```

Comment:
We still need to embed the format details in the name.  We'd want something similar to https://github.com/coreos/coreos-assembler/blob/de9791c896163bf2e27cf858f6481393951f0557/src/cmd-generate-release-meta#L188-L193 producing an artifact name like `secex.qcow2.xz`.